pkg/workflows/management/create_install_eksa.go

```diff
@@ -37,6 +37,38 @@ func (s *installEksaComponentsOnBootstrapTask) Checkpoint() *task.CompletedTask
 	return nil
 }
 
+type installEksaComponentsOnWorkloadTask struct{}
```

Comment:
this task confuses me
it says "install" components
but it's actually creating the cluster

IMO these should be two separate tasks

1. install eksa crds and controllers
2. create the cluster